src/core/BufferAttribute.js

```diff
@@ -310,6 +311,14 @@ BufferAttribute.prototype = {
 
 		return new this.constructor().copy( this );
 
+	},
+
+	discard: function () {
+
+		var oldArray = this.array;
+
+		this.array = new oldArray.constructor( 1 ); // create dummy minimal length TypedArray
```

Comment:
`discard()` may confuse users. One could think that `discard()` is going to remove the data from the GPU (as that's how the other discard()s work).

How about moving this code to your app instead? So you would do...

```js
attribute.onUpload( function () {

    this.array = new ( this.array.constructor )( 1 );

} );
```